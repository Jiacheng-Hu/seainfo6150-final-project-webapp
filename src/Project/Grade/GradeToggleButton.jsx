import React from "react";

const GradeToggleButton = props => {

    return (
        <button onClick={props.onClick} className="btn btn-default  btn-sm">{props.buttonText}</button>
    )
}

export default GradeToggleButton