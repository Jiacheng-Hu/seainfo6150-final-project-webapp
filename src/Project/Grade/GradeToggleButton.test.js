import React from "react";
import GradeToggleButton from "./GradeToggleButton";


describe("GradeItem tests", () => {
    it("renders correctly", () => {
        const content = "Show More";
        const {container} = render(<GradeToggleButton buttonText={content}/>);
        expect(container).toMatchSnapshot();
    });
});