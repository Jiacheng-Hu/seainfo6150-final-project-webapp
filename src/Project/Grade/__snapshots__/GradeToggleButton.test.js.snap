// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GradeItem tests renders correctly 1`] = `
<div>
  <button
    class="btn btn-default  btn-sm"
  >
    Show More
  </button>
</div>
`;
